components/tabs/src/storage.rs

```diff
@@ -23,15 +23,15 @@ pub struct ClientRemoteTabs {
 
 pub struct TabsStorage {
     local_id: String,
-    local_tabs: Option<Vec<RemoteTab>>,
+    local_tabs: RefCell<Option<Vec<RemoteTab>>>,
```

Comment:
I had to make this a `RefCell` (and `clone` in `get_locaL_tabs`) because `wipe` needs to mutate this field to `None`.